hugegraph-core/src/main/java/com/baidu/hugegraph/schema/IndexLabel.java

```diff
@@ -108,7 +108,7 @@ public void indexField(Id id) {
     }
 
     public Id indexField() {
-        E.checkState(this.indexType == IndexType.RANGE ||
+        E.checkState(this.indexType.isRange() ||
```

Comment:
use this.indexType.isSearch(), and check other `this.indexType == IndexType.SEARCH`